core/src/main/java/com/datastax/oss/driver/internal/core/metadata/DefaultTopologyMonitor.java

```diff
@@ -277,8 +273,7 @@ private NodeInfo asNodeInfo(AdminRow row) {
    *     from {@link #findInPeers(AdminResult, InetSocketAddress)}. Callers that don't already have
    *     it can use {@link #getBroadcastRpcAddress}.
    */
-  protected DefaultNodeInfo.Builder nodeInfoBuilder(
```

Comment:
I prefer to return the object fully built since this method was always called with `nodeInfoBuilder(...).build()`. This way we can avoid returning a concrete type. (Note: we will have to modify the DSE driver accordingly.)